Program/internal_exam2.md

```diff
@@ -0,0 +1,112 @@
+### PROGRAM NO:-8 {Create table cricketer(cid, cname, match, run) and insert 8 players records. Print player records with average.  and 
+### write this data into player.csv file. Do all this task from python}.
+
+
+```python
+import csv
```

Comment:
Is there any other way to import csv for reading and writing ??

Reply:
Yes , we can use other ways to import csv for reading and writing
one of them is by  importing pandas library
example:
df = pd.read_csv('file.csv')